articles/cms/wordpress/troubleshoot.md

```diff
@@ -1,57 +1,74 @@
-# WordPress Troubleshoot
+# WordPress Troubleshooting
 
-## My configuration is wrong and I can't authenticate using Auth0. Is there another way to access the plugin?
+### My configuration is wrong and I can't authenticate using Auth0. Is there another way to access the plugin?
 
 The plugin can be accessed using valid WordPress credentials through the regular WordPress login by adding `?wle` to the login url. For example: `http://yourdomain.com/wp-login.php?wle`.
 
-## I am having problems when a user logs in. Where can I find a log of what is happening?
+### I am having problems when a user logs in. Where can I find a log of what is happening?
 
-The plugin provides an error log where you can check what has happened. Access it through the **Error Log** sub-item of the **Auth0** menu. 
+The plugin provides an error log where you can check what has happened. Access it through the **Error Log** sub-item of the **Auth0** plugin menu.
 
-## How can I redirect the users to a certain URL after login
+### When using a plugin to force the login, the user is not logged in.
```

Comment:
there are plugins that force the user to be authenticated to access the site, they are used to improve the wordpress access control.